app/views/bourbons/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>List of Bourbons</h1>
+<ul>
+<% @bourbons.each do |bourbon| %>
+<li>
```

Comment:
Indentation is a little off here. Indent the `<li>` tag so you can tell at a glance that it is nested inside the `<ul>`.